tools/tests/cmd/collect.trycmd

```diff
@@ -26,6 +26,9 @@ DAP Task Parameters:
           
           [env: HPKE_PRIVATE_KEY=]
 
+      --hpke-config-json <HPKE_CONFIG_JSON>
```

Comment:
How about `--hpke-key-json`? For those in the know, "HPKE config" should only refer to the public part, while for the uninitiated, the word "key" indicates that the file is crypto-related. On the other hand, the control plane and CLI do talk about HPKE configs.